### PR TITLE
News: updates and corrections

### PR DESCRIPTION
**Pull request by grahamperrin**
Round one. 

Remove 'bsdtalk' <https://bsdtalk.blogspot.com/>. The most recent recording is more than five years old. 

Podcast (singular). 

DaemonForums, not Daemon Forums. https, not http. 

Remove 'FreeBSD Diary'. The most recent entry is more than a decade old. Instead: 'Dan Langille's Other Diary', which I loosely describe as a 'successor'. 

Reword the bullet point for 'BSD Now'

Partial background: [Friday question - FreeBSD related podcasts?](https://www.reddit.com/r/freebsd/comments/z4cyns/friday_question_freebsd_related_podcasts/)

### CODE REVIEW

File: website/content/en/news/_index.adoc [hunk 1]
```diff
@@ -13,14 +13,13 @@ loadNews: false
...
 
-* https://www.bsdnow.tv/[BSD Now]: BSD Now is a weekly video podcast which covers the latest news, interviews and tutorials from the world of BSD.
-* https://bsdtalk.blogspot.com/[bsdtalk]: bsdtalk is an audio podcast with interesting interviews of people in the BSD community.
+* https://www.bsdnow.tv/[BSD Now] (weekly) covers the latest news, interviews and tutorials in the world of BSD.
 
 == Other sites
 
-* http://www.daemonforums.org/[Daemon Forums]: Active online forums and news community site dedicated to FreeBSD and other BSDs.
-* https://www.freebsddiary.org/[FreeBSD Diary]: One man's record of his trials and triumphs with FreeBSD.
+* https://dan.langille.org/[Dan Langille's Other Diary]: successor to _The FreeBSD Diary_, which provided practical examples between 1988 and 2012. Dan is, https://www.langille.org/[amongst other things], a committer to the FreeBSD ports tree.
```
Comment (grahamperrin) [inline website/content/en/news/_index.adoc:22]: @dlangille, for the purpose of <https://www.freebsd.org/news/>, would you agree with my use of the word _successor_?

Comment (grahamperrin) [reply website/content/en/news/_index.adoc:22]: Whilst there's no longer a direct link to (old) _The FreeBSD Diary_, readers should easily find their way – if required – via the _amongst other things_ link.

Comment (dlangille) [reply website/content/en/news/_index.adoc:22]: Sure. Thanks.

**Comment by grahamperrin** [comment]
[⚙ D37562 News: updates and corrections](https://reviews.freebsd.org/D37562)

**Review by pauamma — COMMENTED** [review]
WFM.

**Comment by grahamperrin** [comment]
Phoronix informed, with reference to <https://github.com/freebsd/freebsd-doc/commit/b6020386c77e1c7a6e77144a4aa4309060fec9a3?diff=split>.

**Pull request closed without merging**